packages/calling-stateful-client/src/Converter.ts

```diff
@@ -108,6 +108,18 @@ export function convertSdkCallToDeclarativeCall(call: CallCommon): CallState {
     declarativeRemoteParticipants[toFlatCommunicationIdentifier(participant.identifier)] =
       convertSdkParticipantToDeclarativeParticipant(participant);
   });
+  /* @conditional-compile-remove(hide-attendee-name) */
+  // TODO: Replace this once the SDK supports hide attendee name
+  const hideAttendeeNames = false;
+  /* @conditional-compile-remove(hide-attendee-name) */
+  if (hideAttendeeNames) {
+    call.remoteParticipants.forEach((participant: SdkRemoteParticipant) => {
+      const declarativeParticipant = call.remoteParticipants[toFlatCommunicationIdentifier(participant.identifier)];
+      if (declarativeParticipant.role === 'Attendee') {
+        declarativeParticipant.displayName = participant.role;
```

Comment:
We can't set displayName directly to the role as the role isn't localized, instead the UI should map the role to a localized version

Reply:
Yeah, that was left by mistake. I have moved the logic to selectors(Video gallery selector & Participant List selector) but still unsure of how to pass locale in selectors.